councilmatic_core/templates/search/search.html

```diff
@@ -315,10 +322,10 @@ <h3 class="modal-links">
                 var addtl_filter = $(this).attr('data');
 
                 if(existing_q){
-                    window.location.assign('/search/?'+existing_q+'&selected_facets='+encodeURIComponent(addtl_filter)+"&_="+timestamp);
```

Comment:
@evz - is there a reason for these timestamps? I may have been a little too casual in removing them.

Reply:
I actually do not see any use of these timestamps. So, confidently removing them.